src/technology-consulting/5-communication.md

```diff
@@ -9,23 +9,52 @@
 
-- <input type="checkbox"/> Wiki - _GitHub Wiki_
-- <input type="checkbox"/> Screen Sharing - _Google Meet_ or _Slack Calls_
-- <input type="checkbox"/> Video Conference - _Google Meet_ or _Slack Calls_
-- <input type="checkbox"/> Chat - _Slack_
-- <input type="checkbox"/> Calendars - _Shared Google Calendars_ and a project/company calendar.
-- <input type="checkbox"/> Meetings - When meetings are held.
-
+- Wiki - _GitHub Wiki_
+- Screen Sharing - _Google Meet_ or _Slack Calls_
+- Video Conference - _Google Meet_ or _Slack Calls_
+- Chat - _Slack_
+- Calendars - _Shared Google Calendars_ and a project/company calendar.
+- Meetings - When meetings are held.
 
 ## The fundamentals of proactive communication
 
 The following best practices might seem basic, but surprisingly few people follow through with them. Simply performing “the basics” will make you stand out, ensure clarity and reduce risk of an expectation gap growing over time.
 
-### Successful meetings
+### Remote Work
+
+Remote work adds additional challenges to an already difficult job. As a consultant you want to optimize for value and since your customer can't observe your work you have to communicate it.
+
+#### Always be moving
+
+Not a single day should go by without some sort of observable progress. Let's say you have a feature or bug that takes multiple days to complete. It shouldn't be radio silence until the task is completed. You want to walk the line between being communicative and annoying.
+
+For example, when you start the work you change the status of the ticket and then open a pull request and mark it as Work in Progress (WIP). Be sure to push your work up at the end of each day. You may even reach out to your customer or a team member just to go over your work and make sure they independantly understand the ticket the same way.
+
+#### Tell before being asked
+
+If your customer has to ask you what that status of a ticket is, outside of a standup, you need to communicate more. Your customer is your customer, not your manager. You gain and keep confidence by lining up your progress with their expected progress.
+
+Never feel like you are bothering your client by sending them a gif or short movie every day, showing your progress.
+
+#### State expecations explicitly
+
+Communication is hard, and that is why it is one of the longest sections of this training. It is even harder to coordinate while not in the same room. When communicating with your customer, but really this applies to any coordinated communcation; always state your understanding of expecations explicitly.
+
+If you are off in your understanding or re-iteration the other person may be annoyed, but they will be even more annoyed if a large amount of time is spent working in the wrong direction.
```

Comment:
I think this can be a little more clear about what to do, but basically, restate the expectations/goals/assignments, both in words (for immediate confirmation) and in writing (for documentation later).